packages/react-notifications/src/components/notifications-list/notifications-list.tsx

```diff
@@ -30,199 +30,209 @@ type Props = {
   domain: TAppNotificationDomain;
 };
 
-const NotificationsList = (props: Props) => {
+const NotificationsListGlobal = (props: Props) => {
```

Comment:
Ah kay. My mind was expecting `GlobalNotificationsList` and `PageNotificationsList` but that's kind of the same.